src/services/housing.js

```diff
@@ -78,23 +84,27 @@ const getApplicationID = async (username) => {
  * @param {Number} applicationID the application ID number if it is known, else it is -1
  * @param {String} editorUsername the student username of the person filling out the application
  * @param {ApartmentApplicant[]} applicants Array of ApartmentApplicant objects
- * @param {ApartmentChoice[]} preferredHalls Array of ApartmentChoice objects
+ * @param {ApartmentChoice[]} apartmentChoices Array of ApartmentChoice objects
  * @return {Promise.<Number>} Application's ID number
  */
 const saveApartmentApplication = async (
   applicationID,
   editorUsername,
   applicants,
-  preferredHalls,
+  apartmentChoices,
 ) => {
   let applicationDetails = {
     AprtAppID: applicationID,
     Username: editorUsername,
     Applicants: applicants.map((applicant) => applicant.Profile.AD_Username),
     // Applicants: applicants.map((applicant) => { Username: applicant.Profile.AD_Username, OffCampusProgram: applicant.OffCampusProgram }); // This is the correct code for when the backend has been updated expect the off-campus program info
-    ApartmentChoices: preferredHalls,
+    ApartmentChoices: apartmentChoices,
   };
-  return await http.post(`housing/apartment/save/`, applicationDetails);
+  if (applicationID === -1) {
+    return await http.post(`housing/apartment/save/`, applicationDetails);
+  } else {
+    return await http.put(`housing/apartment/save/`, applicationDetails);
```

Comment:
I assume this is temporary since the if and else both do the same thing.

Reply:
They do not do the same thing, one is "post" and the other is "put", which on the backend side directs them to different methods of the HousingController